### PR TITLE
Adding the gtest submodule and the README.md file

### PR DESCRIPTION
**Pull request by SgStapleton**

### CODE REVIEW

**Comment by SgStapleton** [comment]
The CMakeList.txt file will need to be updated to make use of the gtest submodule once that is in

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+git push -u origin master
+```
+4. `mkdir build && cd build`
+5. `cmake .. && make` (TODO This command will need to be updated)
+6. `ctest`
+
+---
+
+## Building without a package manager
+TODO Do we still need this section at all...?
```
Comment (jessemapel) [inline README.md]: This section can go away.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+After you've set up conda, you can build SpiceRefactor:
+
+1. Fork `USGS-Astrogeology/SpiceRefactor` if you don't already have a fork.
+2. Clone your fork of `SpiceRefactor` *with `--recursive` option to get the gtest submodule*.
+```bash
+git clone --recursive git@github.com:<your-username>/SpiceRefactor.git
+cd SpiceRefactor
+git remote add upstream git@github.com:USGS-Astrogeology/SpiceRefactor.git
+```
+3. Sync your fork with `upstream` and ensure the gtest submodule is init'd if your fork is old.
```
Comment (jessemapel) [inline README.md]: I think we can get rid of the bit about how to fork and clone the repo

File: README.md (hunk 1 above)
Comment (kberryUSGS) [inline README.md]: I don't think we need this section. Will we even support this?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### Setting up conda for tcsh
+Copy and paste the following into a terminal running the `tcsh` shell:
+```tcsh
+echo  "\n\n# Setting up miniconda3 for tcsh" >> $HOME/.cshrc && \
+echo  "source $HOME/miniconda3/etc/profile.d/conda.csh > /dev/null" >> $HOME/.cshrc && \
+source $HOME/.cshrc && \
+which conda
+```
+> *For more information: [tcsh installation](https://github.com/ESMValGroup/ESMValTool/issues/301 "Reference to tcsh conda install")*
```
Comment (jessemapel) [inline README.md]: Can we link to an Anaconda install guide instead of writing the instructions out here?

**Comment by kberryUSGS** [comment]
This will fix #1

**Pull request merged**